index.js

```diff
@@ -364,15 +364,19 @@ class ServerlessS3Local {
     return (
       this.service &&
       this.service.plugins &&
-      this.service.plugins.indexOf('serverless-plugin-additional-stacks') >= 0
+      this.service.plugins.modules
+			? this.service.plugins.modules.indexOf('serverless-plugin-additional-stacks') >= 0
+			: this.service.plugins.indexOf('serverless-plugin-additional-stacks') >= 0
     );
   }
 
   hasExistingS3Plugin() {
     return (
         this.service &&
         this.service.plugins &&
-        this.service.plugins.indexOf('serverless-plugin-existing-s3') >= 0
+        this.service.plugins.modules
```

Comment:
In the following modifications, Is 'serverless-plugin-additional-stacks' correct ?
I think that 'serverless-plugin-existing-s3' should be here.